Makefile

```diff
@@ -1,109 +1,7 @@
-# Minimal makefile for Sphinx documentation
-#
+# This Makefile stub enables making starter pack targets.
+# Replace this comment with your project's Makefile contents,
+# overriding any SP-specific targets that clash with your general targets,
+# e.g. install, help or run
```

Comment:
Not everybody is familiar with Makefiles - can we be a bit clearer here?
The idea is that this Makefile provides access to the SP-specific targets defined in the other Makefile, and that you can include your own (not starter pack related) targets in here, or define different target names for the starter pack targets if they clash.
I don't think I would understand that from the current comment.